src/components/message/message-card-list/MessagesVirtualizedList.tsx

```diff
@@ -101,7 +101,11 @@ const MessagesVirtualizedList = (props: Props) => {
 	]);
 
 	const debouncedScrollHandler = useDebouncedCallback(
-		(event: React.UIEvent<'div'>, wheelScrollDirection?: 'next' | 'previous') => {
+		(
```

Comment:
i wonder why does scroll event event triggers when message is scrolled.
can we try rewriting it completely

There is a scrollerRef in Virtuoso

I think if rewrite it this way it would be easier to understand what's happening

```
useEffect(() => {
	function onScroll() {}
	function onWheel() {}

	scrollerRef.addEventListener('scroll', onScroll)
	scrollerRef.addEventListener('wheel', onWheel)
} ,[loadNext, loadPrev])

```